target/scripts/helpers/utils.sh

```diff
@@ -17,9 +17,44 @@ function _escape_for_sed() {
 # Returns input after filtering out lines that are:
 # empty, white-space, comments (`#` as the first non-whitespace character)
 function _get_valid_lines_from_file() {
+  _convert_crlf_to_lf_if_necessary "${1}"
+  _append_final_newline_if_missing "${1}"
```

Comment:
Yeah, if you have a suggestion to change it to a pipe instead that only transforms the text being parsed into lines instead of needing to write it, I'm all good for that 👍

Reply:
Minimal effort: Rename the function to `_fix_file_and_get_content`